### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by abhinavdangeti**

### CODE REVIEW

File: modules/fts/pages/fts-index-analyzers.adoc [hunk 1]
```diff
@@ -12,20 +12,23 @@ An analyzer consists of modules, each of which performs a particular, sequenced
...
-By means of the common token `enjoy`, this permits a match between `enjoyed` and `enjoyable`.
+For example, if the input-text for a search is `liked staying here`, and the document-content contains the phrase `many things to like`, the dictionary-based words do not permit a match.
+However, by using an analyzer (_en_ for this example) that _tokenizes_ and _stems_ words by means of its inner _Tokenizer_ and _Token Filter_ components, the input-text yields the tokens `like` and `stai`; while the document-content yields the tokens `mani`, `thing` and `like`.
+By means of the common token `like`, this permits a match between `like` and `liked`.
 
 Since different analyzers pre-process text in different ways, effective Full Text Search depends on the right choice of the analyzer, for the type of matches that are desired.
 
 Couchbase Full Text Search provides a number of pre-constructed analyzers that can be used with Full Text Indexes.
-Additionally, analyzers can be custom-created by means of the Couchbase Web Console.
+Additionally, analyzers can be custom-created on the Couchbase Web Console or using the http://bleveanalysis.couchbase.com/analysis[Text Analysis Wizard].
```
Comment (sreekanth-cb) [inline modules/fts/pages/fts-index-analyzers.adoc:24]: http://bleveanalysis.couchbase.com/analysis => how easy it is integrate the exported json content of the custom analyser with that of an existing/new index definition?

IMO, it's very hard and error-prune.  So, not sure of mentioning it from a creation perspective.

Rather - if the bleve analysis tool had an import option to pass in a analysis JSONsection from a custom created analyzer of an index definition, then the user would be able to test it easily.?  (feature for bleve analysis tool)

File: modules/fts/pages/fts-index-analyzers.adoc [hunk 2]
```diff
@@ -12,20 +12,23 @@ An analyzer consists of modules, each of which performs a particular, sequenced
...
+
 The remainder of this page explains the architecture of analyzers and describes the modular components that Couchbase Full Text Search makes available for custom-creation.
 It also lists the pre-constructed analyzers that are available and describes the modules that they contain.
 
-For examples of both selecting and custom-creating analyzers by means of the Couchbase Web Console.
+Also, here's a relevant article on https://blog.couchbase.com/full-text_search_text_analysis/[Text Analysis within a Full-Text Search Engine].
```
Comment (sreekanth-cb) [inline modules/fts/pages/fts-index-analyzers.adoc:29]: Just a follow-up doubt - Is this (embedding blog links) a recommended practice for the documentation? 
tagging @RayOffiah and @jon-strabala.

File: modules/fts/pages/fts-index-analyzers.adoc (hunk 1 above)
Comment (abhinavdangeti) [reply modules/fts/pages/fts-index-analyzers.adoc:24]: With the analysis wizard, the user will be able to export a custom analyzer definition. But yes copying that JSON snippet into the index definition can be error prone.

File: modules/fts/pages/fts-index-analyzers.adoc (hunk 2 above)
Comment (abhinavdangeti) [reply modules/fts/pages/fts-index-analyzers.adoc:29]: Ya, will want some advise on this. I embedded this link here per Keshav's recommendation.

**Review by Thejas-bhat — APPROVED** [review]

**Comment by jon-strabala** [comment]
@RayOffiah I have no issue with the blog right now (important to communicate how to do things) but I agree eventually it should be subsumed into the documentation we just don't have the bandwidth right now.

File: modules/fts/pages/fts-index-analyzers.adoc (hunk 2 above)
Comment (jon-strabala) [reply modules/fts/pages/fts-index-analyzers.adoc:29]: Eventually we should clean this up, but for now leave it as is.

**Review by jon-strabala — APPROVED** [review]

**Pull request merged**